native/packages/react-native-bpk-component-carousel/src/__snapshots__/BpkCarousel-test.android.js.snap

```diff
@@ -53,7 +53,8 @@ Array [
   </RCTScrollView>,
   <View
     accessibilityLabel="I'm accessible"
-    accessible={true}
+    childrenSortFn={[Function]}
+    pointerEvents="box-none"
```

Comment:
`pointerEvents` gets added by `TransitionGroup` ... Since we are just rendering dots and nothing within them is tappable I am happy with ignoring this.